### PR TITLE
Feature/a1 mtdc@204 squash flyway

### PR DESCRIPTION
**Pull request by algattik**
Reduced flyway scripts.

Deploy run: https://github.com/catenax/tractusx/actions/runs/1550152459

### CODE REVIEW

File: coreservices/partsrelationshipservice/prs-api/src/main/resources/db/migration/V1_0__part_relationship.sql [hunk 1]
```diff
@@ -4,8 +4,12 @@ CREATE TABLE public.part_relationship (
     parent_oneidmanufacturer character varying(255) NOT NULL,
     parent_objectidmanufacturer character varying(255) NOT NULL,
     part_relationship_list_id uuid NOT NULL,
-    upload_date_time timestamp without time zone NOT NULL
+    upload_date_time timestamp without time zone NOT NULL,
+    effect_time timestamp without time zone DEFAULT CURRENT_TIMESTAMP(0) NOT NULL,
+    removed boolean DEFAULT false NOT NULL,
+    life_cycle_stage character varying(255) DEFAULT 'BUILD'::character varying NOT NULL,
```
Comment (cpeeyush) [inline coreservices/partsrelationshipservice/prs-api/src/main/resources/db/migration/V1_0__part_relationship.sql:10]: nit: for what purpose we need '::character varying' ? Asking as this wasn't there before.

**Review by cpeeyush — APPROVED** [review]

File: coreservices/partsrelationshipservice/prs-api/src/main/resources/db/migration/V1_0__part_relationship.sql (hunk 1 above)
Comment (algattik) [reply coreservices/partsrelationshipservice/prs-api/src/main/resources/db/migration/V1_0__part_relationship.sql:10]: I just deployed `main`, ran `pg_dump` and applied what came out :)

An advantage of using `pg_dump` is you can continue doing the same and have clean diffs going forward.

File: coreservices/partsrelationshipservice/prs-api/src/main/resources/db/migration/V1_0__part_relationship.sql [hunk 2]
```diff
@@ -4,8 +4,12 @@ CREATE TABLE public.part_relationship (
     parent_oneidmanufacturer character varying(255) NOT NULL,
     parent_objectidmanufacturer character varying(255) NOT NULL,
     part_relationship_list_id uuid NOT NULL,
-    upload_date_time timestamp without time zone NOT NULL
+    upload_date_time timestamp without time zone NOT NULL,
+    effect_time timestamp without time zone DEFAULT CURRENT_TIMESTAMP(0) NOT NULL,
```
Comment (cpeeyush) [inline coreservices/partsrelationshipservice/prs-api/src/main/resources/db/migration/V1_0__part_relationship.sql:8]: nit: should we keep default value here ? This was done previously in order to support NOT NULL constraint as this column was added at a later stage and existing data wasn't having a value for effectTime.

Comment (algattik) [reply coreservices/partsrelationshipservice/prs-api/src/main/resources/db/migration/V1_0__part_relationship.sql]: makes sense, removed default

**Review by Izzzu — APPROVED** [review]

**Pull request merged**